.codecov.yml

```diff
@@ -8,11 +8,20 @@ coverage:
   precision: 2
   round: down
   range: "70...100"
-
   status:
-    project: yes
-    patch: yes
-    changes: no
+    project:
```

Comment:
Ok, let's report code coverage, but don't fail on it whatever the value is. The maintainer will make a judgment if the current value is ok or not (this takes care of flaky test reports, or cases when we don't want the CI to run fully - e.g. documentation only changes - AppVeyor).